profiler/build/unix/build.mk

```diff
@@ -3,6 +3,15 @@ CXXFLAGS := $(CFLAGS) -std=c++17
 DEFINES += -DIMGUI_IMPL_OPENGL_LOADER_GL3W
 INCLUDES := $(shell pkg-config --cflags glfw3 freetype2 capstone) -I../../../imgui -I../../libs/gl3w
 LIBS := $(shell pkg-config --libs glfw3 freetype2 capstone) -lpthread -ldl
+
+DISPLAY_SERVER := X11
+libwayland_client := $(shell pkg-config --libs --silence-errors wayland-client)
```

Comment:
Selection of target display server has to be more explicit. I have `wayland-client` installed through package dependencies even on headless machines.

Reply:
I figured.  What about checking for `$WAYLAND_DISPLAY`?  This would only work inside a wayland session though, obviously.

Reply:
Maybe there should be an override environment variable, like `TRACY_LINUX_DISPLAY_SERVER` + a check for `$WAYLAND_DISPLAY` if it is unset.

Reply:
Depending only on `WAYLAND_DISPLAY` may have some unforseen consequences (e.g. some tool in the build chain may want to connect to the server (due to reasons), which isn't present, resulting in build failure). The second option looks good, but maybe with a shorter variable name. Make sure it's properly documented.